aquarius/ddo_checker/shacl_schemas/v4/remote_4.0.0.ttl

```diff
@@ -143,6 +143,56 @@ schema:AlgorithmShape
         sh:node schema:ContainerShape ;
         sh:minCount 1;
         sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:consumerParameters;
+        sh:node schema:ConsumerParametersShape ;
+        sh:maxCount 1;
```

Comment:
I believe `consumerParameters` is a list so there can be more than 1 `ConsumerParameter`. So shouldn't this line be removed?